docs/homebrew.md

```diff
@@ -0,0 +1,66 @@
+# Homebrew Setup
```

Comment:
This is the setup part. See `docs/release.md` for how to create a release which is thankfully a lot simpler 😅